frame/evm/src/lib.rs

```diff
@@ -208,6 +208,12 @@ pub mod pallet {
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
 
+		#[pallet::no_default]
+		type RandomnessProvider: frame_support::traits::Randomness<
+			sp_core::H256,
```

Comment:
The runtime implementing this pallet may not what to enabled randomness. (For example, the template runtime which currently has a really dangerous example)
```suggestion
			Option<sp_core::H256>,
```

We can also put this type behind a `feature`, or just allow the random result to be `None`.